### PR TITLE
[MDCT-1868] Section D: Encounter Data 

### PR DESCRIPTION
**Pull request by juliuscaesar**
## Description
Adds Section D: Encounter Data. Closes [MDCT-1868](https://qmacbis.atlassian.net/browse/MDCT-1868)

### How to test
http://localhost:3000/mcpar/plan-level-indicators/encounter-data-report

### Changed Dependencies
N/A

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/forms/mcpar/dedr/dedr.json [hunk 1]
```diff
@@ -1,18 +1,47 @@
...
+          "hint": "Describe the state's standard for timely encounter data submissions used in this program.</br>If reporting frequencies and standards differ by type of encounter within this program, please explain."
+        }
+      },
+      {
+        "id": "dedr-2",
+        "type": "number",
+        "props": {
+          "label": "D1.III.2 Share of encounter data submissions that met state’s timely submission requirements",
+          "hint": "What percent of the plan’s encounter data file submissions (submitted during the reporting period) met state requirements for timely submission?</br>If the state has not yet received any encounter data file submissions for the entire contract period when it submits this report, the state should enter here the percentage of encounter data submissions that were compliant out of the file submissions it has received from the managed care plan for the reporting period.",
+          "mask": "comma-separated"
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dedr/dedr.json:35]: No mask for now. Will be adding a percentage mask later. Applies to `dedr-2` and `dedr-3`.

Also need to make the fields disabled so users cannot enter data.

```suggestion
          "disabled": true
```

**Review by braxex — CHANGES_REQUESTED** [review]
Minor changes requested.

**Review by gmrabian — APPROVED** [review]

File: services/ui-src/src/forms/mcpar/dedr/dedr.json [hunk 2]
```diff
@@ -1,18 +1,45 @@
...
+          "label": "D1.III.1 Definition of timely encounter data submissions",
+          "hint": "Describe the state's standard for timely encounter data submissions used in this program.</br>If reporting frequencies and standards differ by type of encounter within this program, please explain."
+        }
+      },
+      {
+        "id": "dedr-2",
+        "type": "number",
+        "props": {
+          "label": "D1.III.2 Share of encounter data submissions that met state’s timely submission requirements",
+          "hint": "What percent of the plan’s encounter data file submissions (submitted during the reporting period) met state requirements for timely submission?</br>If the state has not yet received any encounter data file submissions for the entire contract period when it submits this report, the state should enter here the percentage of encounter data submissions that were compliant out of the file submissions it has received from the managed care plan for the reporting period.",
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dedr/dedr.json:34]: Trailing comma here makes the JSON invalid and breaks everything. Same thing for the other `hint` below.

**Review by braxex — CHANGES_REQUESTED** [review]
Super close. Everything is breaking because of some commas in `dedr.json`. Sometimes the littlest things are the most important to check run locally before pushing. 😆

**Review by braxex — APPROVED** [review]

**Pull request merged**